.env-dist

```
@@ -120,4 +121,6 @@ E2E_TEST_ACCOUNT_PASSWORD=
 # Monitor Premium features
 PREMIUM_ENABLED=
 # Link to start user on the subscription process. PREMIUM_ENABLED must be set to `true`.
+SUBSCRIBE_PREMIUM_PRODUCT=
+SUBSCRIBE_PREMIUM_PLAN=
```

In case we expand to other countries later, it might be good to distinguish this from their plans:

```suggestion
SUBSCRIBE_PREMIUM_PLAN_US=
```

And we might even want to [align with Relay](https://github.com/mozilla/fx-private-relay/blob/2a589b32818ec68d7660a2b1c5606c874cdb9b7e/privaterelay/settings.py#L344-L350) and name these `PREMIUM_PRODUCT_ID` and `PREMIUM_PLAN_ID_US`, while we're at it.

---

Also, I believe these aren't secret (I think they're exposed in the URL when subscribing), so we could also add working values here.

---

Right they aren't secret (they are passed query params in the URL the user sees when completing payment), however we do need to configure it for each of our environments.

Should we do the one set up for local development as the default? I think that's what we do for other defaults like SERVER_URL.